src/core/Akka/Routing/RoutedActorCell.cs

```diff
@@ -193,7 +194,14 @@ public override void SendMessage(Envelope envelope)
             if (RouterConfig.IsManagementMessage(envelope.Message))
                 base.SendMessage(envelope);
             else
-                Router.Route(envelope.Message, envelope.Sender);
+            {
+                // Bugfix: https://github.com/akkadotnet/akka.net/issues/7247 
+                if(envelope.Message is IScheduledTellMsg scheduledTellMsg)
```

Comment:
This is the real fix here - and I describe the root of problems here on the issue itself https://github.com/akkadotnet/akka.net/issues/7247#issuecomment-2162089219

Reply:
Technically this change means that certain messages will now show up in Phobos traces if they're sent through a router via the scheduler. I am fine with that.